proto/Makefile

```diff
@@ -41,7 +41,7 @@ build/proto.stamp: $(source_files)
 	rm -rf build/proto pkg
 	mkdir -p build/proto
 	# Protobuf generation.
-	for source in $(source_files) ; do protoc -I src "$$source" --go_out=plugins=grpc:build/proto ; done
+	for source in $(source_files) ; do protoc -I src "$$source" --go_out=plugins=grpc:build/proto ; test $$? -eq 0 || exit 1; done
```

Comment:
why not:
```
 	... do protoc -I src "$$source" --go_out=plugins=grpc:build/proto || exit 1; done
```
instead of
```
 	... do protoc -I src "$$source" --go_out=plugins=grpc:build/proto; test $$? -eq 0 || exit 1; done
```

Reply:
that's exactly the kind of code review this PR needed 🙇